apps/studio/components/interfaces/Home/ProjectUsage.tsx

```diff
@@ -129,7 +129,7 @@ const ProjectUsage = () => {
                 yAxisKey="total_rest_requests"
                 onBarClick={(v: unknown) => handleBarClick(v as UsageApiCounts, 'rest')}
                 customDateFormat={datetimeFormat}
-                highlightedValue={sumBy(charts, 'total_rest_requests')}
+                highlightedValue={Intl.NumberFormat("en-US").format(sumBy(charts, 'total_rest_requests'))}
```

Comment:
I'm not sure if Supabase supports localization, if so this should probably be set to the user's locale and not hardcoded to `en-us`

If not, disregard 😄

Reply:
no localization (yet!)

Reply:
Gotcha 👍 it still might be worthwhile moving this somewhere central like a config file (if there's a pattern for this already) so that you don't have to hunt down every instance of this when there is localization, but that's more of a nit 🙂

Reply:
One option is just removing the locale?
![image](https://github.com/supabase/supabase/assets/62911962/cbfc31a5-170f-4b3f-b515-ea3e0f284f14)

Reply:
that makes sense to me: [The implementation's default locale will be used](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#locales_argument) 🙂